Core/src/test/java/com/plotsquared/core/plot/FlagTest.java

```diff
@@ -74,4 +79,73 @@ public void testFlagName() {
         Assertions.assertEquals("use", flagName);
     }
 
+    @Test
+    public void shouldSuccessfullyParseTitleFlagWithTitleSingularAndSubTitleEmpty() {
+        Assertions.assertDoesNotThrow(() -> {
+            var title = PlotTitleFlag.TITLE_FLAG_DEFAULT.parse("\"test\" \"\"").getValue();
+            Assertions.assertEquals("test", title.title());
+            Assertions.assertEquals("", title.subtitle());
+        }, "Should not throw a FlagParseException");
+    }
+
+    @Test
+    public void shouldSuccessfullyParseTitleFlagWithTitleMultipleWordsAndSubTitleEmpty() {
+        Assertions.assertDoesNotThrow(() -> {
+            var title = PlotTitleFlag.TITLE_FLAG_DEFAULT.parse("\"test hello test\" \"\"").getValue();
+            Assertions.assertEquals("test hello test", title.title());
+            Assertions.assertEquals("", title.subtitle());
+        }, "Should not throw a FlagParseException");
+    }
+
+    @Test
+    public void shouldSuccessfullyParseTitleFlagWithTitleMultipleWordsAndSubTitleMultipleWords() {
+        Assertions.assertDoesNotThrow(() -> {
+            var title = PlotTitleFlag.TITLE_FLAG_DEFAULT.parse("\"test hello test\" \"a very long subtitle\"").getValue();
+            Assertions.assertEquals("test hello test", title.title());
+            Assertions.assertEquals("a very long subtitle", title.subtitle());
+        }, "Should not throw a FlagParseException");
+    }
+
+    @Test
+    public void shouldSuccessfullyParseTitleFlagWithTitleEmptyAndSubTitleSingleWord() {
+        Assertions.assertDoesNotThrow(() -> {
+            var title = PlotTitleFlag.TITLE_FLAG_DEFAULT.parse("\"\" \"single\"").getValue();
+            Assertions.assertEquals("", title.title());
+            Assertions.assertEquals("single", title.subtitle());
+        }, "Should not throw a FlagParseException");
+    }
+
+    @Test
+    public void shouldThrowFlagParseExceptionWithOddQuotes() {
+        var exception = Assertions.assertThrows(FlagParseException.class, () -> {
+            PlotTitleFlag.TITLE_FLAG_DEFAULT.parse("title\"");
+        }, "Needs to throw a FlagParseException");
+        Assertions.assertTrue(exception.getErrorMessage() instanceof TranslatableCaption);
+        Assertions.assertEquals(
+                "flags.flag_error_title_invalid_quote_count",
+                ((TranslatableCaption) exception.getErrorMessage()).getKey()
+        );
+        Assertions.assertEquals(1, exception.getTemplates().length);
+        var template = exception.getTemplates()[0];
+        Assertions.assertTrue(template instanceof Template.ComponentTemplate);
+        Assertions.assertEquals("quote_count", ((Template.ComponentTemplate) template).key());
+        Assertions.assertEquals("1",((TextComponent)((Template.ComponentTemplate) template).value()).content());
```

Comment:
Didn't want to add the plaintext serializer for that. Should be fine here as the template content has no styling.